v1/brokers/sqsv2/sqs.go

```diff
@@ -255,6 +255,13 @@ func (b *Broker) consumeOne(sqsReceivedMsgs *ReceivedMessages, taskProcessor ifa
 		sig.RoutingKey = *queueName
 	}
 
+	// Port changes from https://github.com/securitiai/machinery/pull/11 to sqs v2
+	var attrs map[string]*string
+	for key, val := range delivery.Messages[0].Attributes {
+		attrs[key] = aws.String(val)
```

Comment:
@pakistancan we are assigning values to a nil map here. 
This would cause panic when there are elements in `delivery.Messages[0].Attributes` 

Could you please review once.